iiop-openjdk/src/main/java/org/wildfly/iiop/openjdk/service/CorbaORBService.java

```diff
@@ -110,6 +111,8 @@ public void start(StartContext context) throws StartException {
                 properties.setProperty(ORBConstants.SERVER_HOST_PROPERTY, address.getAddress().getHostAddress());
                 properties.setProperty(ORBConstants.SERVER_PORT_PROPERTY, String.valueOf(address.getPort()));
                 properties.setProperty(ORBConstants.PERSISTENT_SERVER_PORT_PROPERTY, String.valueOf(address.getPort()));
+
+                socketBinding.getSocketBindings().getNamedRegistry().registerBinding(ManagedBinding.Factory.createSimpleManagedBinding(socketBinding));
```

Comment:
Nice; I forgot about socketBinding.getSocketBindings(). So there's no reason for anyone injecting from a SocketBinding service to also depend on the SocketBindingManager service. :)